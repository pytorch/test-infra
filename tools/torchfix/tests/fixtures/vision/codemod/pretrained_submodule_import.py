# Check that the codemod works when a model
# imported from a submodule
# i.e. from torchvision.models.resnet instead of
# directly from torchvision.models.

from torchvision import models
backbone = models.resnet.resnet101(pretrained=True, replace_stride_with_dilation=[False, True, True])

from torchvision.models import resnet
backbone = resnet.resnet101(pretrained=True, replace_stride_with_dilation=[False, True, True])

from torchvision.models.resnet import resnet152
resnet152(pretrained=True)
resnet152(pretrained=False)
