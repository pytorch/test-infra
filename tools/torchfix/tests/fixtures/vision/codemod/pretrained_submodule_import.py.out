# Check that the codemod works when a model
# imported from a submodule
# i.e. from torchvision.models.resnet instead of
# directly from torchvision.models.

from torchvision import models
backbone = models.resnet.resnet101(weights=models.ResNet101_Weights.IMAGENET1K_V1, replace_stride_with_dilation=[False, True, True])

from torchvision.models import resnet
backbone = resnet.resnet101(weights=models.ResNet101_Weights.IMAGENET1K_V1, replace_stride_with_dilation=[False, True, True])

from torchvision.models.resnet import resnet152
resnet152(weights=models.ResNet152_Weights.IMAGENET1K_V1)
resnet152(weights=None)
